src/session.ts

```diff
@@ -136,10 +144,10 @@ export class SessionManager implements Middleware {
 
         if (this.powerShellExePath) {
 
-            var bundledModulesPath = path.resolve(__dirname, "../../modules");
+            let bundledModulesPath = path.resolve(__dirname, "../../modules");
 
             if (this.inDevelopmentMode) {
-                var devBundledModulesPath =
+                const devBundledModulesPath =
                     // this.sessionSettings.developer.bundledModulesPath ||
```

Comment:
Same reply as above.

Reply:
I saw this line commented out over the weekend and wasn't sure why I did it...  I think we should probably uncomment that otherwise the setting is useless.

Reply:
Does this fire any neurons - https://github.com/PowerShell/vscode-powershell/commit/5f9cfd6c664caf29c432f9948f5113fe0aef515a

Reply:
Yes, thanks for that!  The setting is being used [here](https://github.com/PowerShell/vscode-powershell/commit/5f9cfd6c664caf29c432f9948f5113fe0aef515a#diff-f3fe9c4fe673e665c8df5914c3b9c9a0R116) so you can safely delete that commented out line

Reply:
Done